### ISSUE TITLE
Viewport controls are unusable

### ISSUE THREAD

**Issue by shrinidhi666** [NeoAxis/NeoAxisEngine#15]
Hi,
I am trying to make some sense of the viewport mouse controls and the whole thing is very unintuitive. Is there any way to change it so it makes sense? I tried to find if there is an option and failed. I might have overlooked it. Any help here would be welcome. Thanks.

**Comment by IvanEfimov** [comment]
Hi,
Please tell what you think is wrong. Even if it's ok now, although at first it seemed unintuitive to you.

**Comment by shrinidhi666** [comment]
Hi Ivan,
Here are some of the issues I found that can be improved I guess 
1:  F key focuses on the object selected, but I couldn't find a way to rotate the viewport camera around the focused object,
2:  A simple way to change the speed of the viewport camera pan/zoom .
Thanks

**Comment by IvanEfimov** [comment]
666,
1. Try hold F key and right mouse button at same time, then move mouse.
2. You can change speeds from project settings in 'Scene Editor' tab. To fast access don't close the window, use Apply button. Also can move the window to work area of the editor (docking), to make the window as usual document window.

**Comment by shrinidhi666** [comment]
1 : The solution works , but not at all useful since we cannot zoom while F and right mouse button pressed. 
2 : Wasting a whole docking area to just access the viewport camera speed controls is a no brainer.

**Comment by IvanEfimov** [comment]
Yeah, it makes sense to improve the management capabilities, we will work on it. Although this may differ from other programs due to the fact that our idea is to add as few controls as possible to the screen.

Please tell more thoughts, ideas. It very useful.

As variant can use auto hide panels with the settings.
Maybe to add sliders near screen labels in right top corner with the ability to hide it.

**Comment by shrinidhi666** [comment]
awesome. I think you can try adding it in the top right corner.
Agree that it's better to have as few UI controls in the viewport as possible. Maybe you can have a shortcut for different set speeds, along with the autohide UI or something like unreal with a tiny dropdown icon on the right corner of the viewport. 

For focus. just an ability to rotate/zoom the camera around the focused object should be more than fine. that way we don't have to keep refocusing by pressing F . maybe alt + right mouse can rotate around the focused object while retaining the current functionality.
